packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/nodeexpress/enrollmentqrcodeintegrationsteps.md

```diff
@@ -0,0 +1,90 @@
+
+#### IdxTransaction response
+
+An example `IdxTransaction` response follows:
+
+```json
+{
+  status: "PENDING",
+  nextStep: {
+    name: "enroll-poll",
+    inputs: [
+    ],
+    authenticator: {
+      contextualData: {
+        qrcode: {
+          method: "embedded",
+          href: "data:image/png;base64,iVBORw0...",
+          type: "image/png",
+        },
+        selectedChannel: "qrcode",
+      },
+      key: "okta_verify",
+      displayName: "Okta Verify",
+    },
+    poll: {
+      required: true,
+      refresh: 4000,
+    },
+  },
+}
+```
+
+The following table describes the most important `IdxTransaction` properties for this step:
+
+| Property       | Example value | Description                                       |
+|----------------|---------------------------------------------------|---------------------------------------------------|
+| `IdxTransaction.status` | `PENDING`  | Status of transaction. A value of `PENDING` indicates Okta is waiting for the user to complete the Okta Verify setup.|
+| `IdxTransaction.nextStep.name` | `enroll-poll` | Name of the next step in the sign-in. A value of `enroll-poll` indicates the app should show the QR Code and poll Okta to determine when the user has completed the Okta Verify setup.  |
+| `IdxTransaction.nextStep.authenticator.contextualData.qrcode.href` | "data:image/png;base64,..." | The QR code base64 encoded PNG image.
+
+A `IdxTransaction.status` value of `PENDING` and `IdxTransaction.nextStep.name` value of `enroll-poll` signifies that Okta is waiting for the user to add their account to Okta Verify. At this time, your app should start polling to determine when the user has completed the Okta Verify account setup.
+
+#### Render the QR Code
+
+The following [Mustache](https://mustache.github.io/) template snippet shows how the sample app displays the QR code.
+
+```html
+<div class="ui segment">
+    <div class="ui fluid image">
+      <img id="authenticator-qr-code" src="{{contextualData.qrcode.href}}" />
+    </div>
+</div>
+```
+
+#### Display page with QR code
+
+Display a page showing the QR code to the user.
+
+<div class="common-image-format">
+
+![Okta Verify option shown in UI](/img/authenticators/authenticators-oktaverify-enroll-qr-code.png)
+
+</div>
+
+### 3: Poll SDK while enrollment is ongoing
+
+`IdxTransaction.status` of `PENDING` and `nextStep.name` equal to `enroll-poll`, indicates that the user needs to complete the challenge in Okta Verify, and your application should begin polling. The polling logic is common across the different flows, and the steps are described in detail in [Polling Okta](#polling-okta).
+
+### 4: Open Okta Verify
```

Comment:
Why isn't there an Install Okta Verify section earlier on, like there is for Google Auth in the other guide? This shouldn't be a step in the enrollment flow.